README.md

```diff
@@ -39,99 +39,82 @@ Ensure you have access to the Internet from VM. Recommended install is through f
 | Windows        | Windows - MSI          | https://docs.fluentd.org/installation/install-by-msi |
 | Gem Install**	 | MacOS & Linux - Gem			 | https://docs.fluentd.org/installation/install-by-gem | 
 
+##### Gem based install
+For a Gem-based install, the Ruby Interpreter must be setup first. You can install the Ruby Interpreter by doing the following:
 
-```text
-** For Gem based install, Ruby Interpreter has to be setup first, following is the recommended process to install Ruby
+1. Install Ruby Version Manager (RVM) outlined in the [RVM documentation]https://rvm.io/rvm/install#installation-explained.
+   * Use the `SUDO` command  for multi-user installation. For more information, see the [RVM troubleshooting documentation](https://rvm.io/support/troubleshooting#sudo).
 
-1. Install Ruby Version Manager (RVM) as described in https://rvm.io/rvm/install#installation-explained, ensure to follow all the onscreen instructions provided to complete the rvm installation
-	* For installation across users a SUDO based install is recommended, the installation is as described in https://rvm.io/support/troubleshooting#sudo
+2. After the RVM installation is complete, execute the command 'rvm -v' to verify.
 
-2. Once rvm installation is complete, verify the RVM installation executing the command 'rvm -v'
+3. Install Ruby v2.7.0 or above with the command `rvm install <ver_num>`, (for example, `rvm install 2.7.5`).
 
-3. Now install ruby v2.7.0 or above executing the command 'rvm install <ver_num>', ex: 'rvm install 2.7.5'
+4. Verify the Ruby installation, execute `ruby -v`, gem installation `gem -v` and `bundler -v` to ensure all the components are intact.
 
-4. Verify the ruby installation, execute 'ruby -v', gem installation 'gem -v' and 'bundler -v' to ensure all the components are intact
+5. Install the FluentD gem with the command `gem install fluentd`.
 
-5. Post completion of Ruby, Gems installation, the environment is ready to further install new gems, execute the following gem install commands one after other to setup the needed ecosystem
+6. After FluentD is successfully installed, install the following plugins.
 
-	'gem install fluentd'
-
-```
-
-After FluentD is successfully installed, the below plugins are required to be installed
-
-````text
+```shell
 gem install fluent-plugin-concat
 gem install fluent-plugin-datadog
 gem install fluent-plugin-jfrog-siem
```

Comment:
isn't this applicable to splunk only or does datadog also use siem plugin ?

Reply:
@shrajfr12  This is applicable to Datadog as well. This plugin is used to get xray violation data